docs/csharp/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md

```diff
@@ -4,512 +4,512 @@ ms.date: 07/20/2015
 ms.assetid: eec32dbb-70ea-4c88-bd27-fa2e34546914
```

Comment:
Applies to all files in this PR:  This looks like a thorough review and update of these topics. Do you think you should update the `ms.date` field for any or all of these files?